plugins/usbdmx/SyncPluginImpl.cpp

```diff
@@ -50,6 +50,8 @@
 #include "plugins/usbdmx/GenericDevice.h"
 #include "plugins/usbdmx/ShowJockeyDMXU1.h"
 #include "plugins/usbdmx/ShowJockeyDMXU1Factory.h"
+#include "plugins/usbdmx/Siudi.h"
+#include "plugins/usbdmx/SiudiFactory.h"
```

Comment:
Are they Sunlite officially and just rebadged as others? If so we should probably call this SunliteSIUDI.*

Reply:
Here is what I found: Nicolaudie in France seems to be the ventor. They have this brand called sunlite. See here for example: `https://pdf.archiexpo.com/pdf/nicolaudie-europe/catalog-developer-kits-siudi-6c/54341-176345.html` and here `https://www.audiolight.sg/product-brand/sunlite/`.

It seems that firmware development was done by `LightingSoft AG` in Switzerland. I think this an incorrect entry in the usb ID list.

Long story short: It is a „Sunlite Intelligent Usb Dmx Interface“, short SIUDI made by Nicolaudie.

SunliteSIUDI would be bad as the S in SIUDI already stands for Sunlite. Better NicolaudieSIUDI.

Reply:
I left it as `Siudi.cpp` as other drivers seem to have only the product name in their file names.